### PR TITLE
Feature: Strapi media library integration.

### PR DESCRIPTION
**Pull request by kbinieda**

### CODE REVIEW

File: admin/src/components/CKEditorInput/plugins/StrapiMediaLib.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const Plugin = window.CKEditor5.core.Plugin;
```
Comment (kbinieda) [inline admin/src/components/CKEditorInput/plugins/StrapiMediaLib.js:1]: @pomek Is there a better way to import ck5 classes for dll mode than using window element? It seems a little bit dirty :)

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "@ckeditor/ckeditor5-basic-styles": "^35.1.0",
     "@ckeditor/ckeditor5-block-quote": "^35.1.0",
     "@ckeditor/ckeditor5-code-block": "^35.1.0",
+    "@ckeditor/ckeditor5-core": "^35.1.0",
```
Comment (Reinmar) [inline package.json:33]: To be removed. It's bundled into `ckeditor5`.

File: package.json [hunk 3]
```diff
@@ -53,6 +54,7 @@
     "@ckeditor/ckeditor5-special-characters": "^35.1.0",
     "@ckeditor/ckeditor5-table": "^35.1.0",
     "@ckeditor/ckeditor5-theme-lark": "^35.1.0",
+    "@ckeditor/ckeditor5-ui": "^35.1.0",
```
Comment (Reinmar) [inline package.json:57]: To be removed. It's bundled into `ckeditor5`.

File: admin/src/components/CKEditorInput/index.js [hunk 4]
```diff
@@ -33,6 +34,25 @@ const CKEditorInput = ({
...
+
+    assets.map(asset => {
+      if (asset.mime.includes('image')) {
+        const imgTag = `<p><img src="${asset.url}" alt="${asset.alt}"></img></p>`;
+
+        newValue = `${newValue}${imgTag}`
+      }
+    });
+
+    onChange({ target: { name, value: newValue } });
```
Comment (Reinmar) [inline admin/src/components/CKEditorInput/index.js:52]: Unfortunately this is... wrong. AFAICS this replaces the entire editor content with the old content + the images.

This is neither optimal, nor insert these images at the caret position.

The idea would be to use this command: [https://ckeditor.com/docs/ckeditor5/latest/api/module_image_image_insertimagecommand-InsertImageCommand.html](https://ckeditor.com/docs/ckeditor5/latest/api/module_image_image_insertimagecommand-InsertImageCommand.html)  
  
But I failed to connect this callback with the editor instance. Probably due to my poor React exp.

Comment (Reinmar) [reply admin/src/components/CKEditorInput/index.js]: Doh, had to use the hook obviously. It works now.

**Pull request merged**